scholia/app/templates/base.html

```diff
@@ -84,457 +90,40 @@
-                                               language=language);
-                         $('#wembedder-result-' + listed_q).empty();
-                         $('#wembedder-result-' + listed_q).text(label);
-                     }
-		 });
-	     });
-	     $( '#wembedder' ).append( '<hr>' );
-	 }
-     });
+	// this query opens the Wikidata item as a different aspect
+	wikiItemAsDifferentAspect(`{% include 'base_different-aspect.sparql' %}`, 
+							  "{{ q }}", 
+							  "{{ url_for('app.index') }}")
+
+    // this query opens a subpage for the Wikidata item, e.g. as cito subpage
+	wikiItemSubpage(`{% include 'base_aspect-subpage.sparql' %}`, 
+	 			     "{{ q }}", 
+				   	 "{{ url_for('app.index') }}")
+
+    /* Wembedder */
+    wembedder("{{ q }}")
+
   {% endif %}
 
  $(document).ready(function () {
-     let search_text = ""
-     $('#searchterm').select2({
-         allowClear: true,
-         {% if request.path.endswith("search") %}
-         placeholder: "Scientist, paper, topic, DOI, etc.",
-         {% else %}
-         placeholder: "Search...",
-         {% endif %}
-         ajax: {
-             url: "https://www.wikidata.org/w/api.php?callback=?",
-             dataType: 'jsonp',
-             data: function (params) {
-                 search_text = params.term
-                 var query = {
-                     search: params.term,
-                     action: "wbsearchentities",
-                     language: "en",
-                     uselang: "en",
-                     format: "json",
-                     strictlanguage: true
-                 }
-                 return query
-             },
-             processResults: function (data) {
-                 var results = [{id: -1, text: "Advanced search"}];
-                 $.each(data.search, function (index, item) {
-                     results.push({
-                         id: "{{ url_for('app.index') }}" + item.title,
-                         text: item.label + " - " + item.description
-                     });
-                 });
-
-                 return {
-                     "results": results
-                 };
-             }
-         }
-     });
-
-     $("#searchterm").on("select2:select", function (e) {
-         const search_id = e.params.data.id
-         if (search_id === -1) {
-             if(search_text === undefined) search_text = ""
-             window.location.href = "/search?q=" + search_text;
-         } else {
-             window.location.href = e.params.data.id;
-         }
-     });
-
-     // Add anchor links to all headings
-     var headers = document.querySelectorAll('h2[id], h3[id]')
+	 searchTerm(placeholder=`{{ 
+		 "Scientist, paper, topic, DOI, etc." if request.path.endswith("search") else "Search ..."
+		}}`,
+		urlfor="{{ url_for('app.index') }}")
+    
+	// Add anchor links to all headings
+	var headers = document.querySelectorAll('h2[id], h3[id]')
```

Comment:
@carlinmack What do you want me to do with this code, leave it here or move it to the sholia.js file like the others?